lib/views/header-bar.js

```diff
@@ -84,22 +83,6 @@ class HeaderBarView extends React.Component {
     </TouchableOpacity>
   }
 
-  renderHamburgerButton() {
-    var visible = !this.props.expandedMode;
-    if (!visible) return;
-
-    debug('render hamburger button');
-
-    return <TouchableOpacity
-      testId="hamburger-button"
-      style={styles.hamburger}
-      onPress={this.props.onHamburgerPressed.bind(this)}>
-      <Image
-        style={styles.hamburgerImage}
-        source={require('../images/header-bar-hamburger.png')}/>
-    </TouchableOpacity>
-  }
-
```

Comment:
But it's in version control history! Not a fan of keeping dead code around fwiw.